packages/phrases/src/locales/en.ts

```diff
@@ -268,6 +268,7 @@ const translation = {
         web: 'Web',
         native: 'Native',
       },
+      add_multi_platform: ' supports multiple platform, select a platform to continue',
```

Comment:
nit: Change comma to period. `platform. Select a platform to continue`

Reply:
Aha, I'll leave this to @fleuraly